### PR TITLE
stone: update

### PR DESCRIPTION
**Pull request by ghost**
Redmi Note 12 5g / Poco X5 5g  (Sunstone+Moonstone=Stone)

### CODE REVIEW

File: stone.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Rohitsinh",
+        "oem": "Xiaomi",
+        "device": "Redmi Note 12 5g/Poco X5 5g",
```
Comment (gwolf2u) [inline stone.json]: try decreasing device name

File: stone.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "filename": "crDroidAndroid-13.0-20231121-stone-v9.11.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/stone/9.11/crDroidAndroid-13.0-20231121-stone-v9.11.zip/download",
```
Comment (gwolf2u) [inline stone.json]: use generated download url

File: stone.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "forum": "https://xdaforums.com/t/rom-13-stone-crdroid-v9-11-official-21-11-2023.4642262/",
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Elite-Releases/crdroid-official/NikGapps-T/07-Nov-2023/NikGapps-crdroid-official-arm64-13-20231107-signed.zip/download",
+        "firmware": "https://xiaomifirmwareupdater.com/firmware/sunstone/",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://sourceforge.net/projects/stoneroms/files/Stone-Recovery/OrangeFox-Unofficial-stone.img/download",
+        "paypal": "https://www.paypal.me/ROHITSINDHAV2510",
+        "telegram": "https://telegram.me/rohitsinhprjkttalks",
+        "dt": "https://github.com/crdroidandroid/android_device_xiaomi_stone",
+        "common-dt": "https://github.com/crdroidandroid/android_device_xiaomi_sm6375-common",
```
Comment (gwolf2u) [inline stone.json:24]: please correct commit history
https://github.com/crdroidandroid/android_device_xiaomi_sm6375-common/commit/524d37aeeb1f32249d5b1a22cb6c22223d2f5fe4
https://github.com/crdroidandroid/android_device_xiaomi_sm6375-common/commit/bbc0b45fb827aaddee83154ba3c5d248f0f214be

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: stone.json (hunk 3 above)
Comment (ghost) [reply stone.json:24]: This is fixed

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**